tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -430,6 +430,23 @@ def config_bridge_domain(agent, test_bd, stepinfo='bridge-domain config:')
   end
 end
 
+# Helper for setting system bridge-domain configs
```

Comment:
Agreed. Go ahead. It looks like dependency_manifest() didn't take this into account. I'll start a new thread on that issue.